JS/Lesson34/goksel/recipe-app/src/modules/user/RegistrationForm/index.jsx

```diff
@@ -0,0 +1,139 @@
+import { useState, useEffect } from 'react';
+import './styles.css';
+import { Link } from 'react-router-dom';
+
+export const RegistrationForm = ({ setIsSignedIn, isSignedIn }) => {
+	const initialFormState = {
+		username: '',
+		email: '',
+		phoneNumber: '',
+		password: '',
+	};
+
+	const initialFormErrorsState = {
+		username: null,
+		email: null,
+		phoneNumber: null,
+		password: null,
+	};
+
+	const [formData, setFormData] = useState(initialFormState);
```

Comment:
Use react-hook-form to handle form data in an easier way.

Reply:
Used react-hook-form